lib/core/scheduler.rb

```diff
@@ -151,6 +155,51 @@ def configuration_triggered?(connector_settings)
       false
     end
 
+    def filtering_validation_triggered?(connector_settings)
+      return false unless connector_registered?(connector_settings.service_type)
+
+      filtering = connector_settings.filtering
+
+      unless filtering.present?
+        Utility::Logger.info("#{connector_settings.formatted} does not contain filtering to be validated.")
+
+        return false
+      end
+
+      draft_filters = filtering[:draft]
+
+      unless draft_filters.present?
+        Utility::Logger.info("#{connector_settings.formatted} does not contain a draft filter to be validated.")
+
+        return false
+      end
+
+      advanced_filter_config = draft_filters[:advanced_config]
+
+      # rules checking will be added with future work
+      unless advanced_filter_config.present?
```

Comment:
If there had previously been:
```
"advanced_snippet": {
  "some": "value"
},
"validation":{
  "state": "valid"
}
```

and now we have:
```
"advanced_snippet": {},
"validation":{
  "state": "edited"
  }
  ```
  
  we want this to fall through, right? And not return false here?